satpy/tests/reader_tests/test_agri_l1.py

```diff
@@ -390,6 +370,7 @@ def test_agri_for_one_resolution(self, resolution_to_test, satname):
         available_datasets = reader.available_dataset_ids
         band_names = CHANNELS_BY_RESOLUTION[resolution_to_test]
         self._assert_which_channels_are_loaded(available_datasets, band_names, resolution_to_test)
+        # band_names = ["C07"]
```

Comment:
Is this a reminder of the contents of band_names or a forgotten comment?

Reply:
Forgotten comment. I'll get rid of it.